opt/group_load.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
This is likely to break on systems other than yours. A more portable way is to use `#!/usr/bin/env bash`

https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash

Also, give a newline after this please.

Reply:
I have changed the shebang line. Where should I add the instructions in the README? Should I create a new heading? If so, what should it be called?